wagtail/search/backends/database/sqlite/sqlite.py

```diff
@@ -668,7 +664,10 @@ class SQLiteSearchBackend(BaseSearchBackend):
     def __init__(self, params):
         super().__init__(params)
         self.index_name = params.get("INDEX", "default")
-        self.config = params.get("SEARCH_CONFIG")
+
+        # SQLite backend currently has no config options
+        self.config = None
+        self.autocomplete_config = None
```

Comment:
This might be a sign that we need to extract a `BaseDatabaseSearchBackend` to share common attributes/methods for the database search backends, but I'm happy with these being set individually for now.